tests/test_unasync.py

```diff
@@ -53,13 +71,15 @@ def test_build_py(tmpdir):
     env["PYTHONPATH"] = os.path.realpath(os.path.join(TEST_DIR, ".."))
     subprocess.check_call(["python", "setup.py", "build"], cwd=pkg_dir, env=env)
 
-    unasynced = os.path.join(pkg_dir, "build/lib/example_pkg/_sync/__init__.py")
+    unasynced = os.path.join(
+        pkg_dir, "build", "lib", "example_pkg", "_sync", "__init__.py"
+    )
```

Comment:
So that wasn't the issue after all? Since you don't need this in the test_build_py_modules test

Reply:
I will update that.

Reply:
And sorry for making too many commits.

Reply:
No worries! It's easy to squash when merging